client/dist/style.css

```diff
@@ -3,7 +3,7 @@ body {
 }
```

Comment:
Just wondering if we are supposed to be using a .css file at all. From my understanding that may cause styling conflicts, but you can always refactor into styled-components/css modules if thats the case!

Reply:
yes, thanks for the advice, still working on adding functionality, will do a final reactor after finish all the functionality adding.